src/server/string_family.cc

```diff
@@ -976,7 +976,7 @@ void StringFamily::IncrBy(CmdArgList args, ConnectionContext* cntx) {
   int64_t val;
 
   if (!absl::SimpleAtoi(sval, &val)) {
-    return (*cntx)->SendError(kInvalidIntErr);
+    return cntx->SendError(kInvalidIntErr);
```

Comment:
i know it looks fragile but memcache parser already does all the parsing checks because those conditions should not trigger with memcache.